File: client-web/src/app/core/models/contact-message.models.ts
export enum ContactMessageType {
  Feedback = 0,
  Complaint = 1,
  Query = 2
}

export enum ContactMessageStatus {
  Open = 0,
  Resolved = 1
}

export interface MyContactMessage {
  id: string;
  type: ContactMessageType;
  message: string;
  imageUrl?: string;
  status: ContactMessageStatus;
  response?: string;
  createdAtUtc: string;
}

export interface CreateContactMessageRequest {
  type: ContactMessageType;
  message: string;
  imageUrl?: string;
}

File: LaundryMgmt.Mobile/Models/ContactMessageModels.cs
namespace LaundryMgmt.Mobile.Models;

public enum ContactMessageType
{
    Feedback = 0,
    Complaint = 1,
    Query = 2
}

public enum ContactMessageStatus
{
    Open = 0,
    Resolved = 1
}

public record MyContactMessageDto(
    Guid Id, ContactMessageType Type, string Message, string? ImageUrl,
    ContactMessageStatus Status, string? Response, DateTimeOffset CreatedAtUtc);

public record CreateContactMessageRequest(ContactMessageType Type, string Message, string? ImageUrl);
